doc/doxygen/DoxygenLayout.xml

```diff
@@ -6,6 +6,7 @@
     <tab type="user" visible="yes" title="Tutorial" url="@ref Tutorial" />
     <tab type="user" visible="yes" title="Code gallery" url="@ref CodeGallery" />
     <tab type="modules" visible="yes" title="" intro=""/>
+    <tab type="topics" visible="yes" title="" intro=""/>
```

Comment:
Wouldn't you want to remove the `modules` tab? Or do you intend to be backward compatible? If the latter, I'd prefer a comment after line 8 that says that this is only necessary for doxygen pre-1.9.8.